elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -49,24 +51,36 @@
 
     private static MetricRegistry metricRegistry = null;
     private static HealthCheckRegistry healthCheckRegistry = null;
+    private ElideDynamicEntityCompiler dynamicEntityCompiler = null;
 
     /**
-     * Constructor
+     * Constructor.
      *
-     * @param injector Injection instance for application
+     * @param injector Injection instance for application.
      */
     @Inject
     public ElideResourceConfig(ServiceLocator injector, @Context ServletContext servletContext) {
         this.injector = injector;
 
         settings = (ElideStandaloneSettings) servletContext.getAttribute(ELIDE_STANDALONE_SETTINGS_ATTR);
 
+        dynamicEntityCompiler = new ElideDynamicEntityCompiler(settings.getDynamicConfigPath());
```

Comment:
This piece of code is duplicated between Util.getEntityManager and here. Instead this could be a separate method in Util like initialize dynamic compiler which takes a boolean argument derived from enableDynamicModelConfig . And based on the boolean value and if dynamicEntityCompiler is null, dynamicEntityCompiler will be initialized and classes compiled.